src/hotspot/cpu/aarch64/interpreterRT_aarch64.cpp

```diff
@@ -267,7 +267,7 @@ class SlowSignatureHandler
 
   virtual void pass_object() {
     intptr_t* addr = single_slot_addr();
-    intptr_t value = *addr == 0 ? NULL : (intptr_t)addr;
+    intptr_t value = *addr == 0 ? nullptr : (intptr_t)addr;
```

Comment:
This doesn't compile - perhaps replace nullptr with zero? Unless casting it is more appropriate.

Reply:
There's similar code in other patches, I did casting in those.